a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -54,11 +56,18 @@ def execute(self, listeners, health_mon, vthunder, flavor=None):
                 flavors.update(parsed_exprs)
                 args.update({'monitor': flavors})
 
+        try:
+            health_mon.type = openstack_mappings.hm_type(self.axapi_client, health_mon.type)
```

Comment:
Not sure is it ok to overwirte health_mon.type or not.

Reply:
We are not overwriting here, we are checking if the mapping and exception have been handled to give an appropriate error message to user.

Reply:
+1 on the above comment. This is correct usage.